opendebates_emails/tasks.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from celery import shared_task
+
+from opendebates.models import Submission
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def send_email_task(type, idea_pk):
+    # Avoid circular import:
+    from opendebates_emails.models import EmailTemplate
+
+    logger.debug("send_email_task(%s, %s)", type, idea_pk)
+    try:
+        try:
+            template = EmailTemplate.objects.filter(type=type).order_by("?")[0]
```

Comment:
Yeah, who knows how long before the task runs.  Not that it'd be a big deal if this just raised the exception and the outer try/except caught it.